docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started
+---
+
+In this guide we will create a Kubernetes cluster in Docker, using a containerized version of Talos.
+
+Running Talos in Docker is intended to be used in CI pipelines, and local testing when you need a quick and easy cluster.
+Furthermore, if you are running Talos in production, it provides an excellent way for developers to develop against the same version of Talos.
+
+## Requirements
+
+The follow are requirements for running Talos in Docker:
+
+- firecracker, built from `master`
```

Comment:
you can probably expand a bit this section with links from https://github.com/talos-systems/talos/tree/master/internal/pkg/provision/providers/firecracker#firecracker-talos-provisioner